pkg/types/manifest.go

```diff
@@ -32,6 +32,7 @@ type VersionManifest struct {
 	TokensERC1155     *ManifestEntry `json:"tokens-erc1155"`
 	TokensERC20ERC721 *ManifestEntry `json:"tokens-erc20-erc721"`
 	Signer            *ManifestEntry `json:"signer"`
+	TezosSigner       *ManifestEntry `json:"tezossigner"`
```

Comment:
What are your thoughts on running signer service instead of using manifest?
In our case, this is a service that should start by default and work for signing transactions (as ff-signer).
An advanced user will be able to replace it with the new one with a configured key management system that suits his goals.

Reply:
Got it. Pointed the image explicitly instead of using FF manifest.
Thanks @nguyer  for your opinion!